ceph_deploy/osd.py

```diff
@@ -379,7 +379,7 @@ def activate(args, cfg):
         time.sleep(5)
         catch_osd_errors(distro.conn, distro.conn.logger, args)
 
-        if distro.is_el:
+        if distro.is_el and distro.init != 'systemd':
             system.enable_service(distro.conn)
```

Comment:
oh, can you make this an else enable ceph.target ?  this was one of the last things on my list for the basic fc22 support